features/telemetry/common/src/main/resources/telemetry.proto

```diff
@@ -1,15 +1,18 @@
-option java_package = "org.opennms.netmgt.telemetry.ipc";
+syntax = "proto3";
```

Comment:
A couple things here:
1) We need to keep this in proto2 if we're targeting the fix to foundation-2019 or 25.x - we can't break backwards compatibility
2) We shouldn't need to serialize the `isFlowMessage` or `useRoutingKey` fields. These are used to make the decision, but don't need to be in the payload once that's done - I think the overall strategy may need to be revised. The SinkModule for the respective types should be the one driving this logic, and not the individual messages.

Reply:
No arguments on either point. I just looked for a way to preserve the flag/behavior from the parser all the way to the Kafka dispatcher.

If it can be done at the sink modules prior to dispatch, then that would be more efficient.

Reply:
Changed to proto2